mmv1/products/firestore/Database.yaml

```diff
@@ -175,6 +175,7 @@ properties:
   - !ruby/object:Api::Type::Enum
     name: type
     required: true
+    immutable: true
```

Comment:
I agree with all the other changes, but I'm divided about this one. Doing this will prevent database mode changes via Terraform.

That's a very minor complaint, since you're only allowed to do that for empty databases anyways, so delete-then-recreate is just an annoyance rather than a real hindrance. If you think that's an acceptable tradeoff, feel free to resolve.

Reply:
Very good point, thanks Bob. Dropping this change.